tests/providers/azure/e2e.py

```diff
@@ -144,6 +144,10 @@ def testDiskCopyToOtherZone(self):
     operation = self.az.compute.compute_client.disks.begin_delete(
         disk_copy.resource_group_name, disk_copy.name)
     operation.wait()
+    # Delete operation takes some time to propagate within Azure, which in
+    # some cases causes the next test to fail. Therefore we wait a minute
+    # before moving on to the next test.
+    sleep(60)
```

Comment:
If we can't make this more elegant (actually a loop that checks if it's removed or not) I am good with this.

Reply:
Elegant-ified :)

Reply:
nice and thanks!